### PR TITLE
feat(web): Web footer links to support page

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Web footer links to support page

## What

* Web footer links to support page now
* Changed the layout for the bottom left links
* Put the support page link behind a feature flag

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by siggifv — APPROVED** [review]

File: apps/web/layouts/main.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ import { MenuTabsContext } from '../context/MenuTabsContext/MenuTabsContext'
 import { useI18n } from '../i18n'
 import { GET_ALERT_BANNER_QUERY } from '../screens/queries/AlertBanner'
 import { environment } from '../environments'
-import { useNamespace } from '../hooks'
+import { useFeatureFlag, useNamespace } from '../hooks'
```
Comment (kristofer-lp8f) [inline apps/web/layouts/main.tsx:48]: Isn't this available in a module?

**Review by kristofer-lp8f — APPROVED** [review]
LGTM 👍

File: apps/web/layouts/main.tsx (hunk 1 above)
Comment (RunarVestmann) [reply apps/web/layouts/main.tsx:48]: This is the specific hook for the web project which behaves differently from the global useFeatureFlag hook, if that's what you are thinking about.

Comment (RunarVestmann) [reply apps/web/layouts/main.tsx:48]: Or perhaps you are talking about the import being @island.is/web/components instead of this relative path :)

**Pull request merged**